### PR TITLE
Review (General): ID Types

### PR DESCRIPTION
**Pull request by KellyVance**
Should this be _Queensland_ ID Types?
General review please.

### CODE REVIEW

**Comment by DavidCrosswellGSQ** [comment]
> Should this be _Queensland_ ID Types?
Maybe Party Identifier Type?

File: vocabularies/identifier-type.ttl [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix idt: <http://linked.data.gov.au/def/identifier-type/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
```
Comment (DavidCrosswellGSQ) [inline vocabularies/identifier-type.ttl]: Add @prefix sdo: <https://schema.org/> .

File: vocabularies/identifier-type.ttl [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+<http://linked.data.gov.au/def/identifier-type> a owl:Ontology .
+
+idt:conceptScheme a skos:ConceptScheme ;
+ dct:created "2020-03-24T14:56:46+10:00"^^xsd:dateTime ;
+ dct:creator <http://linked.data.gov.au/org/gsq> ;
+ dct:modified "2020-03-24T14:57:04+10:00"^^xsd:dateTime ;
+ dct:source "Compiled by the Geological Survey of Queensland." ;
+ skos:definition "Types of Identification documents and numbers used in Queensland as acceptable proof of identity for individuals, businesses, and companies."@en ;
+ skos:prefLabel "Identifier Type"@en .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/identifier-type.ttl]: Party Identifier Type?

File: vocabularies/identifier-type.ttl [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+ skos:topConceptOf idt:conceptScheme .
+
+idt:acn a skos:Concept ;
+ skos:altLabel "Australian company number"@en ;
+ skos:definition "Every company in Australia is issued with a unique, nine-digit number when registered. This is an Australian Company Number (ACN) and must be displayed on all company documents. Source:ASIC"@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "ACN"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:australian-visa a skos:Concept ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/identifier-type.ttl]: I don't think this is a party identifier

File: vocabularies/identifier-type.ttl [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+ skos:topConceptOf idt:conceptScheme .
+
+idt:drivers-license a skos:Concept ;
+ skos:altLabel "Drivers Licence"@en ;
+ skos:definition "The official permit required for a person to legally drive a motor vehicle in Australia. Issued by the States and Territories of Australia."@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "Drivers license"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:marriage-certificate a skos:Concept ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/identifier-type.ttl]: Not sure that this is a party identifier

File: vocabularies/identifier-type.ttl [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+ skos:prefLabel "Birth certificate"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:certificate-of-australian-citizenship a skos:Concept ;
+ skos:definition "An Australian citizenship certificate is an official document that shows that a person is an Australian."@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "Certificate of Australian citizenship"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:change-of-name-certificate a skos:Concept ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/identifier-type.ttl]: Not sure that this is a party identifier

**Comment by KellyVance** [comment]
Major revision
Now includes. Valid identification for companies and businesses
Personal ID for registering via [QGOV](https://account.qld.gov.au/help/faq.html#howDoIProvide100PointsOfIdentification)
Personal ID for registering for [MyMinesOnline](https://www.business.qld.gov.au/industries/mining-energy-water/resources/minerals-coal/online-services/myminesonline/register) (allows future integration).

Additional collection to distinguish which ID type belongs to which registration system

File: vocabularies/identifier-type.ttl (hunk 1 above)
Comment (KellyVance) [reply vocabularies/identifier-type.ttl]: added

File: vocabularies/identifier-type.ttl (hunk 2 above)
Comment (KellyVance) [reply vocabularies/identifier-type.ttl]: changed

File: vocabularies/identifier-type.ttl (hunk 3 above)
Comment (KellyVance) [reply vocabularies/identifier-type.ttl]: valid QGOV document

File: vocabularies/identifier-type.ttl (hunk 4 above)
Comment (KellyVance) [reply vocabularies/identifier-type.ttl]: valid QGOV document

File: vocabularies/identifier-type.ttl (hunk 5 above)
Comment (KellyVance) [reply vocabularies/identifier-type.ttl]: valid QGOV and MMOL document

**Comment by KellyVance** [comment]
@DavidCrosswellGSQ 
Is the vocabulary intended to be an actual Party Identifier Type list, or is it really a business registration number type list? If its the latter in actuality I can strip this vocab back to bare bones.

**Pull request merged**